changelog.json

```
@@ -1,5 +1,27 @@
 {
   "releases": {
+    "2.4": [
+      "[New] Set Git proxy environment variables from system configuration automatically - #9154",
+      "[Added] Enable creating a GitHub issue from app menu and keyboard shortcut - #8989",
+      "[Added] Add editor support for IntelliJ Idea on macOS - #8398. Thanks @gssbzn!",
+      "[Added] Add editor support for PhpStorm and update WebStorm - #7195. Thanks @dacoto97!",
+      "[Added] Add shell support for Windows Terminal - #7854. Thanks @AndreiMaga!",
+      "[Added] Support keyboard scrolling in diffs - #9074",
+      "[Fixed] Ensure branch protection warning works when branch names contain special characters - #9086",
+      "[Fixed] Ensure intro page is responsive when resizing the window - #9175",
+      "[Fixed] Ensure local branch is also deleted when deleting a branch that no longer exists on the remote - #8831",
+      "[Fixed] Fix graphic indicating files selected in Dark Theme - #8716. Thanks @say25!",
+      "[Fixed] Support selecting text in error dialogs - #7606",
+      "[Fixed] Prevent bringing changes to another branch from failing silently - #9038",
+      "[Fixed] Save the user's position within \"History\" and \"Changes\" tabs when switching - #9061",
+      "[Improved] Update the remote url when a repository's name changes on GitHub - #8590",
+      "[Improved] Allow \"Open in Desktop\" button from pull requests on GitHub to open in a local fork in Desktop - #9012",
+      "[Improved] Support automatically switching between Dark and Light Theme on Windows - #9015. Thanks @say25!",
+      "[Improved] Format errors containing raw Git output with fixed-width font - #8964",
+      "[Improved] Modify \"View on GitHub\" menu item to open upstream for local fork repositories - #9316",
+      "[Improved] Create new branches from the default branch of the upstream repo for forks - #7762",
```

I think when we moved the forks part to the end it made it less clear, but I won't be upset if you don't take this suggestion.
```suggestion
      "[Improved] Create new branches in forks from the default branch of the upstream repo - #7762",
```